projects/dotcms-field-elements/src/components/dot-binary-file/dot-binary-file.tsx

```diff
@@ -0,0 +1,286 @@
+import {
+    Component,
+    Element,
+    Event,
+    EventEmitter,
+    Listen,
+    Method,
+    Prop,
+    State,
+    Watch
+} from '@stencil/core';
+import Fragment from 'stencil-fragment';
+import {
+    DotBinaryTextStatusEvent,
+    DotFieldStatus,
+    DotFieldStatusEvent,
+    DotFieldValueEvent
+} from '../../models';
+import {
+    checkProp,
+    getClassNames,
+    getErrorClass,
+    getHintId,
+    getId,
+    getOriginalStatus,
+    getTagError,
+    getTagHint,
+    isFileAllowed,
+    updateStatus
+} from '../../utils';
+import { Components } from '../../components';
+import DotBinaryTextField = Components.DotBinaryTextField;
+
+/**
+ * Represent a dotcms binary file control.
+ *
+ * @export
+ * @class DotBinaryFile
+ */
+@Component({
+    tag: 'dot-binary-file',
+    styleUrl: 'dot-binary-file.scss'
+})
+export class DotBinaryFileComponent {
+    @Element() el: HTMLElement;
+
+    /** Value specifies the value of the <input> element */
+    @Prop({ mutable: true })
+    value = null;
```

Comment:
How can we pass a value here? is this the identifier of the file? how this works?

Reply:
actually need to remove that `prop` is not needed in this component.